docs/developer-docs/setup/playground.md

```diff
@@ -0,0 +1,279 @@
+- The playground's frontend asset canister, which contains:
+    - A Motoko compiler produced by `js-of-ocaml`. 
+    - A Monaco editor that includes support for Motoko syntax validation.
+    - A Vessel package manager that loads libraries from the vessel package set.
+    - A deploy module which integrates the canister's actor class, Candid UI, and upgrade process. 
+- The playground's backend canister that controls all canisters deployed by users on the playground. 
+
+Each canister has an initial amount of 0.5T cycles and can be used for 20 minutes once the canister has been deployed. To avoid wasting cycles, canisters cannot transfer cycles, and the cycles transfer instructions have been removed.
+
+To ensure resources are used fairly by users, proof of work when a user makes a request for a canister ID is required. 
+
+Since Motoko playground is designed for short, small-scale testing, there are limitations and restrictions imposed on canisters deployed to the playground. These limitations are:
+
+- Cycle transfer instructions are silently ignored by the system.
+- Canisters can use at most 1GB of stable memory.
+- Canisters can call the management canister to manage itself without being the controller.
+- Deployed canisters expire after 20 minutes. Upgrades to canisters reset this timer. When the timer runs out, the canister(s) will be uninstalled. 
+
+## `dfx deploy --playground`
+
+Through `dfx`, canisters can be deployed directly from the CLI to the Motoko playground using the `--playground` flag. When this flag is used, the canisters in the `dfx` project will be deployed to the IC mainnet using borrowed resources from the Motoko playground canister pool, and a canister URL will be returned to the developer. This enables developers to test canister functionality or workflow without needing to set up a cycles wallet, obtain cycles, or configure other parameters that are required for deployment to the mainnet. 
+
+### Prerequisites
+
+Before you start, verify the following:
+
+- [x] You have an internet connection and access to a shell terminal on your local macOS or Linux computer.
+
+- [x] You have a command line interface (CLI) window open. This window is also referred to as the 'terminal' window.
+
+- [x] You have downloaded and installed the IC SDK package as described in the [installing the IC SDK](./install/index.mdx) page.
+
+
+:::info
+`dfx deploy --playground` is available in `dfx` versions v0.15.1 and above.
+:::
+
+To deploy a canister to the Motoko playground, first create a new `dfx` project with the command:
+
+```
+dfx new hello_world
+```
+
+This command will create a new default `dfx` project, which contains a simple 'Hello, world!' sample.
+
+Then, to deploy this canister to the playground, run the command:
+
+```
+dfx deploy --playground
+```
+
+Once deployed, the canister can be interacted with using a command such as:
+
+```
+dfx canister --network playground call hello__world_backend greet '("everyone": text)'
```

Comment:
```suggestion
dfx canister --network playground call hello__world_backend greet '("everyone")'
```
while nice and explicit it's not required to label text, and since this is targeted at extreme beginners I'd drop this to make it less daunting